plutus-ledger-api/src/Plutus/V1/Ledger/Value.hs

```diff
@@ -167,6 +171,19 @@ makeLift ''TokenName
 tokenName :: ByteString -> TokenName
 tokenName = TokenName
 
+-- | A currency, identified by currency symbol and token name.
```

Comment:
I'm not a fan of this name, we've often used `Currency` to mean "all the tokens controlled by a policy". How about `Asset`? or `AssetClass`? given that we've started referring to `(CurrencySymbol, TokenName)` as an "AssetID"?